interface/vchiq_arm/vchiq_test.c

```diff
@@ -1201,15 +1201,15 @@ func_data_test(VCHIQ_SERVICE_HANDLE_T service, int datalen, int align, int serve
    EXPECT(vchiq_queue_message(service, &element, 1), VCHIQ_SUCCESS);
 
    memset(databuf, 0xff, sizeof(databuf));
-   data = (uint8_t *)((uint32_t)databuf & ~(PAGE_SIZE - 1)) + align;
-   data = (uint8_t *)((((int)databuf + PROLOGUE_SIZE) & ~(FUN2_MAX_ALIGN - 1)) + align - PROLOGUE_SIZE);
+   data = (uint8_t *)((unsigned long)databuf & ~(PAGE_SIZE - 1)) + align;
+   data = (uint8_t *)((((long)databuf + PROLOGUE_SIZE) & ~(FUN2_MAX_ALIGN - 1)) + align - PROLOGUE_SIZE);
```

Comment:
Should we be using intptr_t and uintptr_t these days?

http://pubs.opengroup.org/onlinepubs/009695399/basedefs/stdint.h.html